webapp/views.py

```diff
@@ -451,6 +453,24 @@ def advantage_view():
         for account in accounts:
             account["contracts"] = advantage.get_account_contracts(account)
 
+            if account is not None:
+                resp = advantage.get_account_subscriptions_for_marketplace(
+                    account["id"], "canonical-ua"
+                )
+                subs = resp.get("subscriptions")
+                if subs is not None:
+                    subscriptions["current_subscription_number"] = len(subs[0]["purchasedProductListings"])
+                    subscriptions["next_payment"] = {}
```

Comment:
if you initialise `next_payment` at the top too the first subscription does not need a special case I think.